db/crud.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.orm import Session
+from db.models import DBAuthor, DBBook
+from db.schemas import AuthorCreate, BookCreate
+
+
+def get_all_authors(db: Session, skip: int = 0, limit: int = 10):
+    return db.query(DBAuthor).offset(skip).limit(limit).all()
+
+
+def get_author(db: Session, author_id: int):
+    return db.query(DBAuthor).filter(DBAuthor.id == author_id).first()
+
+
+def create_author(db: Session, author: AuthorCreate):
+    db_author = DBAuthor(name=author.name, bio=author.bio)
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
```

Comment:
It's a good practice to wrap the database operations in a try-except block to handle any potential exceptions that might occur during the commit or refresh operations. This will ensure that the application can handle database errors gracefully.